libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/CustomServerUrlEditor.java

```diff
@@ -131,38 +131,7 @@ public void onDismiss(DialogInterface dialog) {
 	public View getRootView() {
 		return rootView;
 	}
-
-	private void setEditText(int editId, String value) {
```

Comment:
Wait - why was this implementation removed? Let's talk offline first. We need this implementation for dynamic server changes.